gef-ui/templates/partials/submit-to-ukef-confirmation.njk

```diff
@@ -9,7 +9,9 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds-from-desktop">
       {{ govukPanel({
-        titleText: "Automatic inclusion notice submitted to UKEF",
+        titleText: "Automatic inclusion notice submitted to UKEF"
+        if isAutomaticCover else 
+        "Manual inclusion application submitted to UKEF",
```

Comment:
If the `submissionType` text is the same in the DB i'd suggest using that instead: 

```js
titleText: deal.submissionType
```

Then it has less conditions in the template and will work for all types AIN/MIA/MIN

Reply:
done